types/es6-promisify/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for es6-promisify 5.0
+// Project: https://github.com/digitaldesignlabs/es6-promisify#readme
+// Definitions by: Harry Shipton <https://github.com/harryshipton>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function es6_promisify(original: (...args: any[]) => any, settings?: Settings): ((...args: any[]) => Promise<any>);
+
+// If the issue at https://github.com/Microsoft/TypeScript/issues/1360 is fixed,
+// then an update should be submitted replacing the above declaration with the
+// following declarations.
+/*
+declare function es6_promisify<T>(original: (...args: any[], callback: (error: any, arg: T) => any) => any, settings?: Settings): ((...args: any[]) => Promise<T>);
+
+declare function es6_promisify(original: (...args: any[], callback: (error: any, ...args: any[]) => any) => any, settings?: Settings): ((...args: any[]) => Promise<any[]>);
+*/
+
+interface Settings {
```

Comment:
Could you wrap this in an `es6_promisify` namespace so that users can access the `Settings` type?